### PR TITLE
Complex analogues

### LINKED ISSUE #34: ComplexInput for analogues_object in wps_rerank
## Description
analogues_object in [`wps_rerank.py`](https://github.com/pacificclimate/chickadee/blob/c238f6ee331e95e161564fed5b51f6e164a13ab5/chickadee/processes/wps_rerank.py#L50) should be ComplexInput with 
```Python
supported_formats=[Format("application/rda", extension=".rda", encoding="base64")]
```

## Requested Action
The process should be able to read files in the user's local directory

## Alternatives
None

### PR DESCRIPTION
**Pull request by cairosanders**
Use complex input for Rdata containing the analogues from CA stage with the same method as used for [quail](https://github.com/pacificclimate/quail/blob/6f6724c1708b614d075b4f15f7af292a982b82de/quail/io.py#L11).

Resolves #34 

Test with `make test-notebooks-custom` on port 30589

### CODE REVIEW

**Review by nikola-rados — APPROVED** [review]
LGTM.

File: chickadee/processes/wps_rerank.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import os
-import re
-from pywps import Process, LiteralInput, ComplexInput, FORMATS
+from rpy2 import robjects
+from pywps.inout.formats import Format
+from pywps import Process, ComplexInput, LiteralInput, FORMATS
```
Comment (sum1lim) [inline chickadee/processes/wps_rerank.py]: You do not have to import `Format` from  `pywps.inout.formats`. It is accessible from `pywps` right away.

File: tests/test_wps_rerank.py [hunk 2]
```diff
@@ -26,6 +26,6 @@ def test_wps_rerank(obs_file, var, qdm_file, analogues_object):
             f"varname={var};"
             f"out_file={out_file.name};"
             f"qdm_file=@xlink:href={qdm_file};"
-            f"analogues_object={analogues_object};"
+            f"analogues_object=@xlink:href={analogues_object};"
```
Comment (sum1lim) [inline tests/test_wps_rerank.py:29]: :+1:

**Review by sum1lim — CHANGES_REQUESTED** [review]
This PR is perfect except that we could reduce a line when importing pakcages.

File: docs/source/notebooks/wps_rerank_demo.ipynb [hunk 3]
```diff
@@ -13,7 +13,7 @@
   },
```
Comment (sum1lim) [inline docs/source/notebooks/wps_rerank_demo.ipynb:13]: <p>:+1:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Review by sum1lim — APPROVED** [review]
Looks good! Merge away.

**Pull request merged**